### DISCUSSION TITLE
Primitive Collisions has no obstacles

### DISCUSSION THREAD

**Discussion by vignesh-anand** [NVlabs/curobo#314] [Software Q&A]
Hi Curobo Team,
I'm trying to initialise and solve IK with curobo. I initialise and then repeatedly update the collision world with solver.update_world() in the sim before planning again. When doing this with an empty world config, I get the exception "Primitive Collision Has no obstacles".  I initialise the planner with collision_checker_type as MESH. I would appreciate some support.

Thanks,
V

**Comment by balakumar-s** [comment]
Do you want collision-free IK? 

Can you provide a small script that reproduces this error?
